holmes-py/specs/gdc_data_portal_v2/cohort_comparison/download_tsv.spec

```diff
@@ -0,0 +1,166 @@
+# Cohort Comparison - Download TSV
+Date Created    : 04/10/2024
+Version			    : 1.0
+Owner		        : GDC QA
+Description		  : Cohort Comparison - Download TSV
+Test-case       : PEAR-816
+
+tags: gdc-data-portal-v2, regression, cohort-comparison
```

Comment:
Can you please add a tag, `cohort-comparison-download-tsv`

Reply:
done